server/src/main/java/com/aws/greengrass/cli/CLIEventStreamAgent.java

```diff
@@ -619,6 +633,20 @@ public ListLocalDeploymentsResponse handleRequest(ListLocalDeploymentsRequest re
             });
         }
 
+        private class DeploymentTimeComparator implements Comparator {
+            @Override
+            public int compare(Object o1, Object o2) {
+                Topics t1 = (Topics) o1;
```

Comment:
Add a check before casting. Make sure it actually is a topics. Or you should be able to use Node I think.